test/bdd/agent/agent_sdk_steps.go

```diff
@@ -184,14 +184,14 @@ func (a *SDKSteps) create(agentID, inboundHosts, inboundPorts, schemes string, o
 	for _, s := range scheme {
 		switch s {
 		case webSocketTransportProvider:
-			inbound, err := ws.NewInbound(schemeAddrMap[s], "ws://"+schemeAddrMap[s])
+			inbound, err := ws.NewInbound(schemeAddrMap[s], "ws://"+schemeAddrMap[s], "", "")
 			if err != nil {
 				return fmt.Errorf("failed to create websocket: %w", err)
 			}
 
 			opts = append(opts, aries.WithInboundTransport(inbound), aries.WithOutboundTransports(ws.NewOutbound()))
 		case httpTransportProvider:
-			opts = append(opts, defaults.WithInboundHTTPAddr(schemeAddrMap[s], "http://"+schemeAddrMap[s]))
+			opts = append(opts, defaults.WithInboundHTTPAddr(schemeAddrMap[s], "http://"+schemeAddrMap[s], "", ""))
```

Comment:
Can we update/add  BDD tests to demonstrate the TLS support ? It would be a good reference for the consumers and also, we can test/confirm https/wss communication between 2 aries agents.

Reply:
Sure. I will create an issue for that. For now, I am preparing the second PR for edge-agent. Which needs this change. It resolves the safari problem.

Reply:
@soluchok which safari problem?

Reply:
@rolsonquadras  https://github.com/hyperledger/aries-framework-go/issues/2117
@troyronda mentioned that `edge-agent` doesn’t work in safari. This PR resolves it. The problem was with unsecured connections. e.g we cannot use `http` and `ws` on the `https` page. 
@llorllale ^

Reply:
@soluchok oh. I heard something more specific - that the agent cannot access the storage *while loaded in the CHAPI frame*. If this solves that issue then great! 

Can you confirm that this is indeed the case?

Reply:
@llorllale I was able to exchange `out-of-band` invitation using `chapi` window through the `didcomm` channel.

Reply:
@sudeshrshetty @troyronda ^^